### PR TITLE
Switch to newer cos-gpu-installer - v2.1.7

### PR DESCRIPTION
**Pull request by dims**
Possible fix for https://github.com/kubernetes/kubernetes/issues/121169

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by dims** [comment]
Looks like there's a 2.1.9 too! https://explore.ggcr.dev/?repo=gcr.io/cos-cloud/cos-gpu-installer

if 2.1.7 does not work then i'll try that

**Comment by dims** [comment]
looking through the commits, it seems safer to go straight to 2.1.9 which is the latest at the moment:
https://cos.googlesource.com/cos/tools/+/41b149e0964426cb38fd469d2f4ff79235ebe9e5%5E%21/#F0

**Comment by dims** [comment]
/skip

looks like the presubmit is broken. let's land and see if the periodic works fine `ci-kubernetes-e2e-gce-device-plugin-gpu`

File: test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml [hunk 1]
```diff
@@ -52,8 +52,8 @@ spec:
         # Refer to details about the installer in https://cos.googlesource.com/cos/tools/+/refs/heads/master/src/cmd/cos_gpu_installer/
         # and the COS release notes (https://cloud.google.com/container-optimized-os/docs/release-notes) to determine version COS GPU installer for a given version of COS.
 
-        # Maps to gcr.io/cos-cloud/cos-gpu-installer:v2.0.27 - suitable for COS M97 as per https://cloud.google.com/container-optimized-os/docs/release-notes
-      - image: gcr.io/cos-cloud/cos-gpu-installer:v2.0.27
+        # Maps to gcr.io/cos-cloud/cos-gpu-installer:v2.1.27 - suitable for COS M109 as per https://cloud.google.com/container-optimized-os/docs/release-notes
```
Comment (pacoxu) [inline test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml:55]:
```suggestion
        # Maps to gcr.io/cos-cloud/cos-gpu-installer:v2.1.7 - suitable for COS M109 as per https://cloud.google.com/container-optimized-os/docs/release-notes
```
Typo in comment

**Pull request merged**